hypervisor/include/arch/x86/asm/boot/ld_sym.h

```diff
@@ -14,5 +14,6 @@ extern uint8_t		ld_entry_end;
 extern const uint8_t	ld_trampoline_load;
 extern uint8_t		ld_trampoline_start;
 extern uint8_t		ld_trampoline_end;
+extern uint8_t		_ld_ram_size;
```

Comment:
Our coding guideline does not recommend variables starting with underscores. See https://projectacrn.github.io/latest/developer-guides/c_coding_guidelines.html#c-id-07-names-defined-by-developers-shall-not-start-with-underscore

Reply:
done